xmind/tests/test_workbook_element.py

```diff
@@ -0,0 +1,751 @@
+import logging
+from . import base
+from unittest.mock import patch, MagicMock, call, Mock
+from xmind.core.workbook import WorkbookElement
+from xmind.core.const import TAG_WORKBOOK, NAMESPACE, XMAP, NS_FO, NS_XHTML, NS_XLINK, NS_SVG, TAG_SHEET, ATTR_VERSION
+
+
+class TestWorkbookElement(base.Base):
+
+    def setUp(self):
```

Comment:
I see patching for methods from ```__init__``` method but I don't see a place where you check that patches were called. Maybe you could introduce method like ```_assert_init_methods_called``` and call it in each test where you need it.